js/packages/eyes-storybook/src/generateConfig.js

```diff
@@ -44,12 +44,35 @@ function generateConfig({argv = {}, defaultConfig = {}, externalConfigParams = [
     result.showLogs = true;
   }
 
-  if (!result.testConcurrency && !result.concurrency) {
-    result.testConcurrency = 5; // TODO require from core
+  result.testConcurrency = utils.types.isInteger(result.testConcurrency)
+    ? result.testConcurrency
+    : utils.types.isInteger(result.concurrency)
+    ? result.concurrency * 5
+    : 5;
+
+  result.serverUrl = result.serverUrl
+    ? result.serverUrl
+    : utils.general.getEnvValue('SERVER_URL') ?? 'https://eyesapi.applitools.com';
+  result.viewportSize = result.viewportSize ? result.viewportSize : {width: 1024, height: 600};
+  result.renderers = result.browser ? result.browser : [{name: 'chrome', width: 1024, height: 768}];
+  result.saveNewTests = result.saveNewTests === undefined ? true : result.saveNewTests;
+  result.keepBatchOpen = result.dontCloseBatches;
+  result.fully = result.fully === undefined ? true : false;
+
+  if (result.batchName) {
+    result.batch = {name: result.batchName, ...result.batch};
+  }
+
+  if (result.batchId) {
+    result.batch = {id: result.batchId, ...result.batch};
   }
 
   if (result.storyDataGap === undefined) {
-    result.storyDataGap = result.testConcurrency;
+    if (result.testConcurrency * 2 >= MAX_DATA_GAP) {
+      result.storyDataGap = MAX_DATA_GAP;
+    } else {
+      result.storyDataGap = result.testConcurrency * 2;
+    }
```

Comment:
What if I have `61` concurrency?
During our discussion, I was envisioning something like this 
```suggestion
    result.storyDataGap = Math.max(Math.min(result.testConcurrency * 2, MAX_DATA_GAP), result.testConcurrency);
```
or
```suggestion
    result.storyDataGap = Math.min(result.testConcurrency * 2, Math.max(MAX_DATA_GAP, result.testConcurrency));
```